### DISCUSSION TITLE
Is there any way to show all lines?

### DISCUSSION THREAD

**Discussion by allenfrostline** [plasma-umass/scalene#516] [Q&A] [answered]
It seems by default the profiling result is filtered and only a subset of lines are shown. Is there any way to avoid this behavior?

**Comment by emeryberger** [comment, marked as answer]
The default behavior has now been changed. The web UI now allows you to toggle a view containing all or only profiled lines.

**Reply by allenfrostline** [reply]
Thanks. Exactly what’s needed 👍🏻
